### PR TITLE
Documentation dashboard page

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #24

### CODE REVIEW

File: docs/src/user/manual/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Dashboard Page User Manual
+
+Welcome to the `Dashboard` page, a centralized hub for a quick overview of linked communities. Follow this user manual to navigate and make the most out of the dashboard experience.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/dashboard.md:3]:
```suggestion
Welcome to the `Dashboard` page, a centralised hub for a quick overview of linked communities. Follow this user manual to navigate and make the most out of the dashboard experience.
```

File: docs/src/user/manual/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+            ![renewal cost of assets](./img/dashboard-6.png)
+
+            1. **Download PDF:** Click on the `Download PDF` option to save the chart as a PDF file.
+
+                ![downloaded pdf](./img/dashboard-7.png)
+
+                ![downloaded pdf](./img/dashboard-8.png)
+
+            2. **Available Options on Chart:** Explore various options on the chart for enhanced visualization:
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/dashboard.md:31]:
```suggestion
            2. **Available Options on Chart:** Explore various options on the chart for enhanced visualisation:
```

File: docs/src/user/manual/dashboard.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+            - **Camera Image (Download as PNG):** Download the plot as a PNG image.
+            
+            - **Magnifying Glass Image:** Click to activate the zooming functionality. Select the bar graph to zoom in for a closer look.
+
+            - **Pan Image:** Click to enable panning. Narrow, widen, slide left, or slide right to adjust the graph view.
+
+            - **Select Box Image:** Activate the select box feature to choose a specific area on the graph.
+
+            - **Lasso Select Image:** Utilize the lasso select image option for precise graph selection.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/dashboard.md:41]:
```suggestion
            - **Lasso Select Image:** Utilise the lasso select image option for precise graph selection.
```

File: docs/src/user/manual/dashboard.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+            - **Plus Image (Zoom In):** Zoom in for a more detailed view of the chart.
+
+            - **Minus Image (Zoom Out):** Zoom out to get a broader perspective of the chart.
+
+            - **Auto Scale Image:** Automatically scale the graph for optimal visualization.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/dashboard.md:47]:
```suggestion
            - **Auto Scale Image:** Automatically scale the graph for optimal visualisation.
```

File: docs/src/user/manual/dashboard.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+            ![maintenance costs of assets](./img/dashboard-9.png)
+
+            ![maintenance costs of assets](./img/dashboard-10.png)
+
+        3. **Annual Average Infrastructure Demand:** This graph uses lifespan projections and renewal costs for a long-term outlook of infrastructure. This projection is theoretical and is not a realistic indication of spending timelines. A valuable output of this projection is an annualized infrastructure demand, indicated as a dotted line on the graph. This annualized value is obtained by dividing the renewal cost by the lifespan for each asset in the database and then summing the total. As lifespan and renewal cost data are updated, the annual infrastructure demand will update. The annual infrastructure demand could be lowered by committing to operations and maintenance programs to extend lifespans, deciding to rehabilitate versus replace, and more. The values shown in the graph are based on current $ values and the actual value of this average annual investment will increase over time with inflation.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/dashboard.md:59]:
```suggestion
        3. **Annual Average Infrastructure Demand:** This graph uses lifespan projections and renewal costs for a long-term outlook of infrastructure. This projection is theoretical and is not a realistic indication of spending timelines. A valuable output of this projection is an annualised infrastructure demand, indicated as a dotted line on the graph. This annualised value is obtained by dividing the renewal cost by the lifespan for each asset in the database and then summing the total. As lifespan and renewal cost data are updated, the annual infrastructure demand will update. The annual infrastructure demand could be lowered by committing to operations and maintenance programs to extend lifespans, deciding to rehabilitate versus replace, and more. The values shown in the graph are based on current $ values and the actual value of this average annual investment will increase over time with inflation.
```

File: docs/src/user/manual/dashboard.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+            ![annual average infrastructure demand](./img/dashboard-11.png)
+
+            ![annual average infrastructure demand](./img/dashboard-12.png)
+
+        4. **Risk By System:** A risk value is obtained by combining Probability of Failure (PoF) and Consequence of Failure (CoF) values as per the following matrix. It is common asset management practice to shift the matrix in favor of the consequence of failure.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/dashboard.md:65]:
```suggestion
        4. **Risk By System:** A risk value is obtained by combining Probability of Failure (PoF) and Consequence of Failure (CoF) values as per the following matrix. It is common asset management practice to shift the matrix in favour of the consequence of failure.
```

File: docs/src/user/manual/dashboard.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+        5. **Remaining Years by Renewal Cost by System:** This section provides insights into the remaining operational years of a system based on its associated renewal cost. Gain a comprehensive understanding of the lifespan and associated costs of various systems within the context of renewal. Use this information for strategic planning and decision-making regarding system maintenance and financial allocation.
+
+            ![remaining years by renewal cost by system](./img/dashboard-15.png)
+
+            ![remaining years by renewal cost by system](./img/dashboard-16.png)
+
+        6. **Remaining Years by Renewal Cost by Risk:** This section offers a detailed analysis of the remaining operational years of a system to its renewal cost and associated risk factors. Understand how the remaining lifespan of different systems correlates with their renewal costs and associated risks. Use this valuable information to prioritize and strategize system maintenance, considering both financial and risk considerations. Enhance decision-making processes by leveraging insights into the intersection of system longevity, renewal costs, and associated risks.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/dashboard.md:77]:
```suggestion
        6. **Remaining Years by Renewal Cost by Risk:** This section offers a detailed analysis of the remaining operational years of a system to its renewal cost and associated risk factors. Understand how the remaining lifespan of different systems correlates with their renewal costs and associated risks. Use this valuable information to prioritise and strategise system maintenance, considering both financial and risk considerations. Enhance decision-making processes by leveraging insights into the intersection of system longevity, renewal costs, and associated risks.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by Jeremy-Prior** [comment]
@ketankartoza please resolve the conflicts

**Comment by ketankartoza** [comment]
> @ketankartoza please resolve the conflicts

@Jeremy-Prior Done!

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Pull request merged**